packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -1862,6 +1864,86 @@ void main() {
     expect(RendererBinding.instance.mouseTracker.debugDeviceActiveCursor(1), SystemMouseCursors.text);
   });
 
+  testWidgets('The allowed interaction is themeable', (WidgetTester tester) async {
+    double value = 0.0;
+
+    Widget buildApp({
+      bool isAllowedInteractionInThemeNull = false,
+      bool isAllowedInteractionInSliderNull = false,
+    }) => MaterialApp(
```

Comment:
```suggestion
    }) => MaterialApp(
```

Please use curly braces `{}`

Reply:
+1